### PR TITLE
Initial modifications for builds

### PR DESCRIPTION
**Pull request by millerts**
This will get more work but this is the initial stuff we'll need for our current builds

### CODE REVIEW

File: install.go [hunk 1]
```diff
@@ -369,6 +379,14 @@ func install(args []string) error {
 
 	// 4. Build and install
 	for _, gom := range goms {
+
+		if install, ok := gom.options["install"].(string); ok {
+			if install != "true" {
+				fmt.Sprintf("Not printing %s", gom.name)
```
Comment (goddardcm) [inline install.go]: "installing" instead of "printing"?

File: install.go [hunk 2]
```diff
@@ -227,6 +236,7 @@ func (gom *Gom) Checkout() error {
 
 func (gom *Gom) Build(args []string) error {
 	installCmd := append([]string{"go", "install"}, args...)
+	//	fmt.Printf("%+v\n", installCmd)
```
Comment (goddardcm) [inline install.go]: remove

File: install.go [hunk 3]
```diff
@@ -174,11 +178,16 @@ func (gom *Gom) pullPrivate(srcdir string) (err error) {
...
+	privateUrl, ok := gom.options["repo"].(string)
+	if !ok {
+		name := strings.Split(gom.name, "/")
+		privateUrl = fmt.Sprintf("git@%s:%s/%s", name[0], name[1], name[2])
+	}
 
 	fmt.Printf("fetching private repo %s\n", gom.name)
+	fmt.Printf("fetching private repo srcdir %s\n", srcdir)
 	cloneCmd := []string{"git", "clone", privateUrl, srcdir}
+	fmt.Printf("%+v", cloneCmd)
```
Comment (goddardcm) [inline install.go]: \n or remove?

File: install.go (hunk 1 above)
Comment (goddardcm) [inline install.go]: and \n

Comment (millerts) [inline install.go]: Not sure why I went with 'printing'.

**Comment by goddardcm** [comment]
:+1:

**Pull request merged**